src/pytestqt/qml/qmlbot.py

```diff
@@ -0,0 +1,41 @@
+from pathlib import Path
+
+from pytestqt.qt_compat import qt_api
+
+
+class QmlBot:
```

Comment:
How about we make this class accessible from `pytestqt` for type annotation, by importing this class in `pytestqt/__init__.py`?

Reply:
Sure